msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1001,6 +1002,23 @@ The `extensions.runtimes` array configures the sets of runtimes and actions that
 
 To use `extensions.runtimes`, see [create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command).
 
+### extensions.contentRuntimes
+
+The `extensions.contentRuntimes` array configures pages of content that are embedded in an Excel or PowerPoint document. 
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`id`| String | 64 characters | ✔️ | Specifies the ID for runtime. Must be unique across all `runtimes` and `contentRuntimes` objects. |
+|`code`| Object | | ✔️ | Specifies the location of content for the runtime. |
+|`code.page`| String | 2048 charaters| ✔️ | The full URL of the page that contains the content that is embedded in the document. |
+|`requestedWidth`| Number | | | The desired width between 32 and 1000 pixels. If this property isn't used, the Office application determines the width.|
+|`requestedHeight`| Number | | | The desired height between 32 and 1000 pixels. If this property isn't used, the Office application determines the height.|
+|`disableSnapshot` | Boolean | | | Specifies whether a snapshot image of your content add-in is saved with the host document. Default value is `false`, so a snapshot is saved. Set to `true` to prevent a snapshot from being saved.|
```

Comment:
Hi @Rick-Kirkham,
Please rephrase the sentence to add 'Default value: `false` ' at the end of the sentence for consistency. Also, in other places, we have mentioned 'host' as 'Office application'. We can follow the same here.